test/mock/services/github_api_mock.dart

```diff
@@ -2,12 +2,20 @@
 import 'dart:io';
 
 import 'package:mockito/mockito.dart';
+import 'package:notredame/core/models/feedback_issue.dart';
 import 'package:notredame/core/services/github_api.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
```

Comment:
Same here (sorry to annoy you with this, and here imports weren't sorted correctly in the first place haha)